docs/src/config_directories.md

```diff
@@ -11,7 +11,7 @@ exist, it will be created.
     [directories]
     cache_dir = "/home/myuser/.tealdeer-cache/"
 
-If no `cache_dir` is specified, tealdeer will fall back to a location that
```

Comment:
Are you sure? So far I thought we have the lowercased word as the "brand name"

Reply:
It's a bit inconsistent 🙈 In the docs, so far we always capitalized the name (except for this line above). In the README, we did not. I don't really mind either way, do you have a preference? I'd slightly tend towards capitalizing it as a proper name (looks less strange at the beginning of a sentence).

Reply:
> I think I would go with lowercase in general, but capitalize at the beginning of a sentence.

Works for me.